cpp/modules/deck.gl/core/src/lib/view-manager.cpp

```diff
@@ -61,16 +61,18 @@ void ViewManager::setNeedsUpdate(const std::string &reason) {
 /** Get a set of viewports for a given width and height
  * TODO - Intention is for deck.gl to autodeduce width and height and drop
  * the need for props
- * @param rect (object, optional) - filter the viewports
- *   + not provided - return all viewports
- *   + {x, y} - only return viewports that contain this pixel
- *   + {x, y, width, height} - only return viewports that overlap with this
- * rectangle
  */
 auto ViewManager::getViewports() -> std::list<std::shared_ptr<Viewport>> {  // {rect) {
   // if (rect) {
   //   return this->_viewports.filter(viewport = > viewport.containsPixel(rect));
   // }
+
+  // TODO(ilija@unfolded.ai): _update isn't being called anywhere, comment for setViews says this:
+  // Does not actually rebuild the `Viewport`s until `getViewports` is called
+  if (this->_needsUpdate.has_value()) {
```

Comment:
Nit: Remove `has_value()` as std::optional has an `operator bool`, allowing us to just write 

```
if (this->_needsUpdate)
```